xsd/benchmark/benchmark-8.xsd

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:tns="https://www.optaplanner.org/xsd/benchmark" elementFormDefault="qualified" targetNamespace="https://www.optaplanner.org/xsd/benchmark" version="1.0">
-
```

Comment:
If you generate the XSD on Java 11 via JAXB, it unfortunately adds unnecessary lines. It does not suffer from this issue on Java 8

Reply:
Do want want to clean up that up manually? or even automatically?
Do those extra lines hurt anyway except for the download size? Few people read the actual XSD.
Proposal: if there's an easy fix in the jaxb configuration, let's do that. If there's not, let's just keep the empty lines, instead of manually (or even automatically) cleaning it up.

Reply:
+1 on keeping the lines. We're a Java 11 project now, so whatever 11 generates is the law.

Reply:
There is no easy fix on the JAXB side. This time I removed the lines as I am releasing the XSD manually. Next time, it will be just updated here with the empty lines.